CHANGES.md

```diff
@@ -33,6 +33,12 @@ To be released.
 
 ### Behavioral changes
 
+ -  `Block<T>.Mine(long, long, Address, HashDigest<SHA256>?, DateTimeOffset,
+    IEnumerable<Transaction<T>>)` became to order `Block<T>.Transactions` by
+    their `Id`.  [[#244], [#355]]
+ -  `Block<T>.Validate(DateTimeOffset)` became to validate
+    if its `Transactions` are ordered by their `Id`.  [[#244], [#355]]
```

Comment:
These changes seems outdated in general.  Need to be updated.

Reply:
As `Block<T>.Validate()` is an `internal` method, this doesn't have to be logged.